### PR TITLE
PSDK-162: Adopt missing LDM dataset props 

### PR DESCRIPTION
**Pull request by jaceksan**
Plus fix titles and tags of generated LDM date dimensions

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/dbt/tables.py [hunk 1]
```diff
@@ -405,9 +444,9 @@ def make_date_datasets(self, table: DbtModelTable, existing_date_datasets: List[
                 date_datasets.append(
                     {
                         "id": column.ldm_id,
-                        "title": self.get_ldm_title(column),
+                        "title": column.gooddata_ldm_title,
                         "description": column.description,
-                        "tags": [table.gooddata_ldm_title] + column.tags,
+                        "tags": ["Date dimensions"],
```
Comment (hkad98) [inline gooddata-dbt/gooddata_dbt/dbt/tables.py:449]: Is this tag by purpose?

File: gooddata-dbt/gooddata_dbt/dbt/tables.py [hunk 2]
```diff
@@ -358,8 +386,9 @@ def make_facts(table: DbtModelTable) -> List[Dict]:
         return facts
 
     @staticmethod
-    def make_labels(table: DbtModelTable, attribute_column: DbtModelColumn) -> List[Dict]:
+    def make_labels(table: DbtModelTable, attribute_column: DbtModelColumn) -> tuple[List[Dict], Optional[dict]]:
```
Comment (hkad98) [inline gooddata-dbt/gooddata_dbt/dbt/tables.py:389]: Please edit
tuple -> Tuple

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/dbt/tables.py (hunk 1 above)
Comment (jaceksan) [reply gooddata-dbt/gooddata_dbt/dbt/tables.py:453]: Yes, unfortunately.
I added a description (from the commit message) here.

File: gooddata-dbt/gooddata_dbt/dbt/tables.py (hunk 2 above)
Comment (jaceksan) [reply gooddata-dbt/gooddata_dbt/dbt/tables.py]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR489

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/284/ : FAILURE in 1m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/434/ : SUCCESS in 1m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/438/ : SUCCESS in 9s

**Pull request merged**